test/bats/label/label.bats

```diff
@@ -0,0 +1,132 @@
+    assert_success
+    echo "$output" > /tmp/bats_test/token
+}
+
+@test "Create Datacenter Label" {
+    datacenter_name="cli-test-datacenter-$(randStr 8)"
+    run ionosctl datacenter create --name "$datacenter_name" -o json 2> /dev/null
+    assert_success
+
+    datacenter_id=$(echo "$output" | jq -r '.id')
+    echo "$datacenter_id"
+
+    assert_output -p "\"name\": \"$datacenter_name\""
+
+    echo "$datacenter_id" > /tmp/bats_test/datacenter_id
+    echo "$datacenter_name" > /tmp/bats_test/datacenter_name
+
+    label_key=$(echo "testlabelkey$(randStr 8)" | tr '[:upper:]' '[:lower:]')
+    label_value=$(echo "testlabelvalue$(randStr 8)" | tr '[:upper:]' '[:lower:]')
+
+    sleep 60
+
+    run ionosctl label add --resource-type datacenter --datacenter-id "$datacenter_id" --label-key "$label_key" --label-value "$label_value" -o json 2> /dev/null
+    assert_success
+    assert_output -p "\"type\": \"label\""
+    assert_output -p "\"key\": \"$label_key\""
+    assert_output -p "\"value\": \"$label_value\""
+
+}
+
+
+
+@test "Create Ipblock Label" {
+    ipblock_name="cli-test-ipblock-$(randStr 8)"
+
+    run ionosctl ipblock create --name "$ipblock_name" -o json 2> /dev/null
+    assert_success
+
+    ipblock_id=$(echo "$output" | jq -r '.id')
+    echo "$ipblock_id" > /tmp/bats_test/ipblock_id
+
+    assert_output -p "\"type\": \"ipblock\""
+
+    sleep 60
+
+    gateway_id=$(echo "$output" | jq -r '.id')
+    label_key=$(echo "testlabelkey$(randStr 8)" | tr '[:upper:]' '[:lower:]')
+    label_value=$(echo "testlabelvalue$(randStr 8)" | tr '[:upper:]' '[:lower:]')
+
+    run ionosctl label add --resource-type ipblock --ipblock-id "$ipblock_id" --label-key "$label_key" --label-value "$label_value" -o json 2> /dev/null
+    assert_success
+    assert_output -p "\"type\": \"label\""
+    assert_output -p "\"key\": \"$label_key\""
+    assert_output -p "\"value\": \"$label_value\""
+}
+
+@test "Delete  at least the two previously created labels" {
+
+      run ionosctl label list --no-headers
+      assert_success
+      initial="$(echo "$output" | wc -l)"
+
+      run ionosctl label remove --all --force
+
+      run ionosctl label list --no-headers
+      new="$(echo "$output" | wc -l)"
+      assert_success
+
+      diff=$(( initial - new ))
+      run [ "$diff" -ge 2 ]
+      assert_success
+
+      run ionosctl ipblock delete --ipblock-id "$(cat /tmp/bats_test/ipblock_id)" -f
+      assert_success
+
+      run ionosctl datacenter delete --datacenter-id "$(cat /tmp/bats_test/datacenter_id)" -f
+      assert_success
+}
+
+teardown_file() {
+    (
+        export IONOS_USERNAME="$(cat /tmp/bats_test/email)"
+        export IONOS_PASSWORD="$(cat /tmp/bats_test/password)"
+
+        ionosctl apigateway gateway delete -af
+    )
+
+    ionosctl user delete --user-id "$(cat /tmp/bats_test/user_id)" -f
+    ionosctl group delete --group-id "$(cat /tmp/bats_test/group_id)" -f
+
+    rm -rf /tmp/bats_test
+}
```

Comment:
Can you please add newline at the end of files? It will fail the commits for people using the pre-commit hooks on our repo

I also recommend installing pre-commit to also run those hooks too https://pre-commit.com/ but it is not necessary